src/pages/manager-review-page/dominant-characteristics/ManagerReviewDominantCharacteristicsExpansionPanel.tsx

```diff
@@ -27,6 +28,10 @@ interface OwnProps {
   reviews: ManagerReviewDominantCharacteristicsExpansionPanel_reviews$key;
   title?: string;
   type: 'strengths' | 'weaknesses';
+  managerPersonReview: {
+    readonly strengths: ReadonlyArray<string> | null;
```

Comment:
the first readonly isn't enough?

Reply:
the first `readonly` is for strengths, you can't change its value i.e `managerPersonReview.strengths = []` and second `readonly` for strengths value that you can't mutate its value.
[for more info](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgLIBM7IN4ChkHJQRzoD2IANgJ7IDOYxIA5mABZ0BcyASieVWoBBKFDjUAPAyihmAPlwBfXLgQUGyALbcMWALw58haRBbsuyANoByAIzWANNYBM1gLpKVmgHQmzHZANLD1wfP1YObwAHAFc6NgAKawAWawBKIA)